src/helpers/emailVerification.js

```diff
@@ -0,0 +1,46 @@
+const nodemailer = require('nodemailer');
+const bunyan = require('bunyan');
+
+module.exports = (username, email, token) => {
+  let transporter = nodemailer.createTransport({
+    service: 'Gmail',
+    auth: {
+      user: 'echo.annafreud@gmail.com',
+      pass: 'FAC!echo1'
+    },
+    logger: bunyan.createLogger({
+      name: 'nodemailer'
+    }),
+    debug: true
+  }, {
+    from: 'Echo do-not-reply <echo.annafreud@gmail.com>',
+  });
+
+  console.log('SMTP Configured');
```

Comment:
Okay, you do not need to run all of this each time you send an email. Put the initialisation at the top level of the module